taskfiles/goreleaser.yml

```diff
@@ -41,4 +41,5 @@ tasks:
     deps:
       - task: :k8s:generate
     cmds:
+      - go version
```

Comment:
Should this be removed?

Reply:
It will be addressed in https://github.com/redpanda-data/redpanda-operator/pull/322